pkg/k8s/synced/crd.go

```diff
@@ -231,25 +226,35 @@ type crdState struct {
 	m map[string]bool
 }
 
-func newCRDState(m map[string]bool) crdState {
+func newCRDState() crdState {
+	m := make(map[string]bool, len(crdResourceNames))
+	for _, name := range crdResourceNames {
+		m[name] = false
+	}
```

Comment:
I think this is fine for explicit-ness, but not necessary. Just wondering, did you run into trouble with this or is this change just for "readability"?

Reply:
Right, was not absolutely necessary, but thought keeping three places where the CRDs were listed "in sync" is unnecessary when can have it done only at one location.

Reply:
So more for "maintainability" :-)